projects/grok/project.yaml

```diff
@@ -0,0 +1,5 @@
+homepage: "https://github.com/GrokImageCompression/grok"
+primary_contact: "boxerab@gmail.com"
+fuzzing_engines:
```

Comment:
I'd recommend removing this section. We'll use libFuzzer, AFL, and honggfuzz by default then.

Reply:
I've removed that, thanks!